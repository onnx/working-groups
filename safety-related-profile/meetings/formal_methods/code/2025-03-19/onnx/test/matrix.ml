let pp_int fmt = Format.fprintf fmt "%2d"
let pp_matrix = Tensor.pretty pp_int

let a = Tensor.matrix [
  [ 1; 2; 3 ];
  [ 4; 5; 6 ];
]

let () = Format.printf " A = %a@." pp_matrix a
let () = Format.printf "tA = %a@." pp_matrix (Tensor.transpose a)
