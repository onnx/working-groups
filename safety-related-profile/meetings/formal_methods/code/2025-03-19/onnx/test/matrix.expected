 A = [  1  2  3 ][  4  5  6 ]
tA = [  1  4 ][  2  5 ][  3  6 ]
